app/views/admin/campaigns/_form.html.haml

```diff
@@ -11,7 +11,7 @@
   .col-md-8
     = semantic_form_for(@campaign, url: (@campaign.new_record? ? admin_conference_campaigns_path : admin_conference_campaign_path(@conference.short_title, @campaign))) do |f|
       = f.inputs do
-        = f.input :name
+        = f.input :name, input_html: { autofocus: true }
```

Comment:
Is the `input_html` needed?  I mean, this should work as well:

``` Ruby
= f.input :name, autofocus: true
```

Doesn't it? 🤔

Reply:
It does not.